### DISCUSSION TITLE
Form.Item tooltip 无法显示

### DISCUSSION THREAD

**Discussion by Aomd** [ant-design/ant-design#36558] [Q&A]
``` tsx
<Form.Item key={index} name={item.fromName} tooltip={{ title: item.label }}>
  {getReactDom(item)}
</Form.Item>
```
tooltip 设置了 trigger 也不显示

"antd": "^4.21.6",

**Comment by valcosmos** [comment]
你好，我试了一下，一般是会正常显示的，但是如果`tooltip`为空的时候，它就不显示了。

```tsx
<Form.Item label="Field A" required tooltip="This is a required field">
  <Input placeholder="input placeholder" />
</Form.Item>
```

这里是对应的文档地址，您可以参考下：

[https://ant.design/components/form-cn/#components-form-demo-required-mark](https://ant.design/components/form-cn/#components-form-demo-required-mark)
